integration_test/sql/migration.exs

```diff
@@ -330,8 +330,36 @@ defmodule Ecto.Integration.MigrationTest do
     end
   end
 
+  defmodule MigrationWithDynamicCommand do
```

Comment:
Same comment as the other one, let's not use integration tests for these. We also need to rename `Dynamic` to `ExecuteFun` or similar.

Reply:
Right, I renamed most of thing, but forgot this one.